lib/Traits/RasterLayerTraits.ts

```diff
@@ -16,4 +50,16 @@ export default class RasterLayerTraits extends ModelTraits {
       "Update a tile only once during this interval when the map is panned. Value should be specified in milliseconds."
   })
   leafletUpdateInterval?: number;
+
+  @objectTrait({
+    type: TileErrorHandlingTraits,
+    name: "tileErrorHandling",
+    description: "Options for handling tile errors"
+  })
+  tileErrorHandlingOptions: TileErrorHandlingTraits = {
+    thresholdBeforeDisablingItem: 0,
```

Comment:
I think  in master the `thresholdBeforeDisablingItem` default is 5, see https://github.com/TerriaJS/terriajs/blob/master/lib/Models/ImageryLayerCatalogItem.js#L203

Reply:
Good catch! I forgot to change it back after testing.